wagtail/test/testapp/fixtures/test_permission_type.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "pk": 10,
+    "model": "wagtailcore.grouppagepermission",
+    "fields": {
+      "group": ["Event moderators"],
+      "page": 12,
+      "permission_type": "bulk_delete"
+    }
+  },
+  {
+    "pk": 11,
+    "model": "wagtailcore.grouppagepermission",
+    "fields": {
+      "group": ["Event moderators"],
+      "page": 12,
+      "permission": ["change_page", "wagtailcore", "page"],
+      "permission_type": "edit"
+    }
+  },
+  {
+    "pk": 12,
+    "model": "wagtailcore.grouppagepermission",
+    "fields": {
+      "group": ["Event editors"],
+      "page": 12,
+      "permission_type": "edit"
+    }
+  }
```

Comment:
This adds three scenarios:
- Old fixture with `permission_type` that's not `"edit"`
- A combination of old and new fixture format, in case someone has this for some reason (e.g. they do `dumpdata`), which should be valid during the deprecation period where both fields exist
- Old fixture with `permission_type` == `"edit"`